Makefile

```diff
@@ -33,7 +33,7 @@ export PYTHONPATH
 
 KWASM_DIR  := .
 KWASM_MAKE := make --directory $(KWASM_SUBMODULE) BUILD_DIR=../../$(BUILD_DIR) RELEASE=$(RELEASE)
-
+export K_OPTS ?= -Xmx16G -Xss512m
```

Comment:
There is already this line in in the kwasm script:

```
export K_OPTS="${K_OPTS:--Xmx16G -Xss512m}"
```

We could set something similar in the Python script if necessary, rather than put it in the Makefile.

Reply:
This options is used in `kompile` rather than `krun`.